### PR TITLE
Update README.md - Add baymax at contributor list

### PR DESCRIPTION
**Pull request by techietrend**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,4 +61,5 @@
 #### [Chivalrousdev](https://github.com/chivalrousdev)
 #### [creative2113](https://github.com/creative2113)
 #### [Virus](https://github.com/gitMan-stack)
```
Comment (BlackGhost2693) [inline README.md:63]: #### [BlackGhost2693](https://github.com/blackghost2693)

File: README.md [hunk 2]
```diff
@@ -61,4 +61,5 @@
...
+#### [Baymax](https://github.com/techietrend)
```
Comment (Ghost1215) [inline README.md:64]: simply

**Review by Ghost1215 — COMMENTED** [review]
perfectly

**Comment by Ghost1215** [comment]
pls resolve the conflict.

**Review by shinevue — APPROVED** [review]
Good solution of conflit

**Pull request merged**